mypyc/lib-rt/CPy.h

```diff
@@ -407,6 +407,13 @@ static int CPy_NoErrOccured(void) {
     return PyErr_Occurred() == NULL;
 }
 
+static void CPy_AssertErrOccurred(void) {
+    assert(PyErr_Occurred() != NULL && "failure w/o err!");
+}
+
+static bool CPy_KeepPropagating(void) {
+    return 0;
+}
```

Comment:
If you define a function in this header, they should be marked `static inline` (we don't do this consistently, but I'd like to make this better).